strr-examiner-web/app/components/DecisionPanel.vue

```diff
@@ -0,0 +1,156 @@
+<script setup lang="ts">
+
+const { t } = useI18n()
+const { showDecisionPanel } = useExaminerDecision()
+const {
+  isApplication,
+  isAssignedToUser,
+  decisionEmailContent,
+  activeReg,
+  activeHeader
+} = storeToRefs(useExaminerStore())
+
+// wip: conditions holder
+const conditions = ref('')
+
+// wip: email content to be sent to Completing Party
+useExaminerStore().decisionEmailContent = t('decision.emailBodyIntro') + '\n\n' + conditions.value
```

Comment:
Can the direct store changes during component setup causes issue? Maybe it can be replaced with a computed.